mxenabled/connect-widget
# copied over changes from the connect repo

#### Issue

https://mxcom.atlassian.net/browse/CT-797

#### Changes

* Added the institution list on searched

#### Testing Instructions

* Open connect and make sure that the search view shows the tiled institutions on the popular list
* Search for an institution and make sure that the view show the institution list.
